src/fides/api/models/privacy_notice.py

```diff
@@ -332,29 +351,152 @@ def create(
         data: dict[str, Any],
         check_name: bool = False,
     ) -> PrivacyNotice:
+        """
+        Creates a Privacy Notice and then a NoticeTranslation record for each supplied Translation.
+
+        For each Translation a historical record is created which combines details from the Notice and
+        the Translation for auditing purposes.  Privacy preferences are saved against the PrivacyNoticeHistory
+        record which contains the full details."""
+        translations = data.pop("translations", []) or []
```

Comment:
PrivacyNotice create method has been updated to also create translations, and then create the PrivacyNoticeHistory record from the combination of the notice and the translation